script/get-failed-run-output.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Mind putting it in `bin`? The only thing we have in `script` should probably be in `.github/scripts`.

You could name it `bin/view-run` and add a `--filter-rspec` flag to tack on the `| grep rspec...`.

Typically I run `gh run` with the grep first and then if nothing comes back, I take off the grep.